### ISSUE TITLE
[FEAT]: last brightness level should be remembered

### ISSUE THREAD

**Issue by alexanderadam** [nova-video-player/aos-AVP#861]
### Description

currently the app forgets the brightness level once it lost the focus.
Switching to another app shortly and going back will lose the brightness information

**Comment by phhusson** [comment]
Makes sense. Maybe limit the timing by one or two minutes? After 2 minutes don't remember the brightness level?

**Comment by alexanderadam** [comment]
In my case I wouldn't would want it to be limited.
I think Netflix also remembers it forever but I'm not sure about this.

**Comment by Pentaphon** [comment]
> Makes sense. Maybe limit the timing by one or two minutes? After 2 minutes don't remember the brightness level?

I think it should be remembered forever.

**Comment by kzssc** [comment]
Sorry for digging this up. But seconding the feature for remembering the brightness

**Comment by Pentaphon** [comment]
@courville any chance you can make this change? I think Nova should do the same thing as other players like Newpipe and remember the brightness setting forever so we don't have to keep changing it each time we open Nova. For example, I prefer having my system brightness at 60% all the time but when I'm watching something on Nova, I'd want it to always be at 90%

**Comment by Pentaphon** [comment]
> Makes sense. Maybe limit the timing by one or two minutes? After 2 minutes don't remember the brightness level?

@phhusson he means always remember the player brightness no matter how many times you open and close the player.
